packages/analytics-nextjs/src/hooks/useAnalytics.ts

```diff
@@ -232,7 +228,7 @@ export function useAnalytics() {
             }
         } else {
             const id = user().id();
-            if (id) {
+            if (id && deferredIdentity?.userId !== id) {
```

Comment:
This was the issue that was reproducing only when AJS cookies were set on `prezly.com`.